core/src/test/scala/reftree/diagram/AnimationSpec.scala

```diff
@@ -1,32 +1,54 @@
 package reftree.diagram
 
+import org.scalatest.prop.PropertyChecks
 import org.scalatest.{FlatSpec, Matchers}
+import reftree.diagram.Animation.Builder
 
-import scala.util.Random
-
-class AnimationSpec extends FlatSpec with Matchers {
+class AnimationSpec extends FlatSpec with PropertyChecks with Matchers {
 
   "iterateUntilFixPoint" should "expand until fixpoint" in {
 
-    val seed = 1337
-    val maxTests = 1200
+    forAll { seed: Int =>
+      whenever(seed > 0) {
```

Comment:
No, i rarely think about performance before I see issues and I didn't recall the syntax for positive ints just now but I'll fix it to match your example (better readability at least ;) )